### PR TITLE
Minor fixes and suggestions.

### PR DESCRIPTION
**Pull request by sherakama**
# READY FOR REVIEW

### CODE REVIEW

File: src/Button/Button.js [hunk 1]
```diff
@@ -1,56 +1,55 @@
...
+import PropTypes from 'prop-types';
+import { buttonVariants, buttonSizes, buttonTypes } from './Button.levers';
 
 /**
  * Button Component
  *
  * HTML button element
  */
-export const Button = ({ className, children, onClick, ref, ...props }) => {
+export const Button = ({ className, children, onClick, ref, variant, size, type, isDisabled, ...props }) => {
```
Comment (sherakama) [inline src/Button/Button.js:10]: If you destructure each of the levers in the constructor you can keep the `{...props}` on the button HTML element which might come in handy. If you don't destructure the props arg then you can't use `{...props}` on the HTML element as React doesn't know what to do with props like `isDisabled: bool`

File: src/Button/Button.js [hunk 2]
```diff
@@ -1,56 +1,55 @@
...
+        break;
 
       case 'minimal':
-        levers.size = classnames('su-p-0')
-        break
+        levers.size = classnames('su-p-0');
+        break;
+
+      default:
+        levers.size = classnames('su-px-26 su-py-10 su-text-20');
```
Comment (sherakama) [inline src/Button/Button.js:52]: Moved default to the default so you don't have to pass in the keyword `default`.

File: src/Button/Button.js [hunk 3]
```diff
@@ -101,4 +100,4 @@ Button.defaultProps = {
   size: 'default',
```
Comment (sherakama) [inline src/Button/Button.js:100]:
```suggestion
```

File: src/Button/Button.levers.js [hunk 4]
```diff
@@ -1,14 +1,14 @@
...
  * Styles for the button.
  */
-export const buttonVariants = ['primary', 'secondary', 'none']
+export const buttonVariants = ['primary', 'secondary', 'none'];
 
 /**
  * Size of the button
  */
-export const buttonSizes = ['default', 'big', 'small', 'minimal']
+export const buttonSizes = ['default', 'big', 'small', 'minimal'];
```
Comment (sherakama) [inline src/Button/Button.levers.js:9]:
```suggestion
export const buttonSizes = ['big', 'small', 'minimal'];
```

**Review by yvonnetangsu — APPROVED** [review]
Cool - thanks!

**Pull request merged**